src/integrationTest/resources/db/migration/V1_3__alter_LRD.sql

```diff
@@ -0,0 +1,3 @@
+
+
+
```

Comment:
Is this class needed - it is empty?

Reply:
yes ...in integration h2 doesn't support sequence and so i just kept empty scripy file otherwise it overrides from src one.